src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -0,0 +1,238 @@
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import useAsync from 'react-use/esm/useAsync';
+
+import {Checkbox, TextInput} from '../../forms/Inputs';
+import Select from '../../forms/Select';
+import {DATATYPES_CHOICES} from './constants';
+import DeleteIcon from '../../DeleteIcon';
+import {PluginsContext} from '../Context';
+import {get} from '../../../../utils/fetch';
+import FAIcon from '../../FAIcon';
+import {ChangelistTableWrapper, HeadColumn} from '../../tables';
+
+const SensitiveData = ({isSensitive}) => {
+  const intl = useIntl();
+  const sensitiveTitle = intl.formatMessage({
+    description: 'Is sensitive icon title',
+    defaultMessage: 'Is sensitive',
+  });
+  const notSensitiveTitle = intl.formatMessage({
+    description: 'Is not sensitive icon title',
+    defaultMessage: 'Is not sensitive',
+  });
+
+  return isSensitive ? (
+    <FAIcon icon="exclamation-circle" title={sensitiveTitle} />
```

Comment:
I think I prefer the variant without circle

Reply:
`fa-exclamation` or `fa-exclamation-triangle`? I find that with the triangle it maybe stands out more :thinking: